src/core_plugins/metrics/public/kbn_vis_types/editor_controller.js

```diff
@@ -1,6 +1,6 @@
 import modules from 'ui/modules';
 import '../services/executor';
-import 'plugins/timelion/directives/refresh_hack';
+import '../../../timelion/public/directives/refresh_hack';
```

Comment:
I would rather not write tests for `refresh_hack` and instead have a separate PR to add a `fetch` event in all the right places. This is something the framework should already provide (which it doesn't).